### DISCUSSION TITLE
Descriptive & aggregate metrics

### DISCUSSION THREAD

**Discussion by BenoitDalFerro** [huggingface/datasets#2394] [Ideas]
Currently there is no basic descriptive statistics metric.

Also the metrics logic is based on an item by item computation and doesn't trivially support aggregate function computation over the entire dataset, e.g. variance

**Comment by mariosasko** [comment]
Hi! We are aware of this issue and are discussing the possible separation of metrics to evolve the API more easily.

**Comment by BenoitDalFerro** [comment]
@mariosasko got a report this issue has been solved, is there any documentation ?

**Comment by mariosasko** [comment]
You should use [`evaluate`](https://github.com/huggingface/evaluate) for metrics now (the metrics API is deprecated in `datasets` and will be removed in the next major release).
